packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/network/ProgressiveStringDecoderTest.kt

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.modules.network
+
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class ProgressiveStringDecoderTest {
+
+  private val TEST_DATA_1_BYTE =
```

Comment:
Can I ask you to use a multiline string literal here and all the other `TEST_DATA_x...` fields?

https://kotlinlang.org/docs/java-to-kotlin-idioms-strings.html#use-multiline-strings

So we won't have to manually concatenate strings with `\n` all over the places.